src/components/scores/concept-coach-cell.cjsx

```diff
@@ -10,15 +10,49 @@ ConceptCoachCell = React.createClass
 
   mixins: [CellStatusMixin] # prop validation
 
+  getInitialState: ->
+    placement: 'right'
+
+  setPopoverPosition: ->
```

Comment:
okies! Do we still need this code to guess whether to appear on the left or the right? It seems to only show up on the left but that seems ok since the student names will always provide enough space to the left of any assignment